docs/user-guide/manage-environments.md

```diff
@@ -178,7 +178,25 @@ To inspect the deployed application in KubeRocketCI portal, take the following s
 
     ![Inspect application](../assets/user-guide/environments/application_terminal.png "Inspect application")
 
+### Delete Application
+
+There are two buttons in the environment which can be used in the portal:
+
+  ![Clean button](../assets/operator-guide/edp-portal-clean-application.png "Clean button")
+
+  * **Delete** - Deletes selected applications and associated resources.
+  * **Clean** - This button is useful when you need to perform custom cleanup actions, e.g. delete cloud resources, database, rollback transactions, etc.
+
+The **Delete** button works best when your application doesn't require deploying spicific resources, such as databases and external applications, or configuring the environment in a specific way.
+
+The **Clean** button triggers a delete pipeline that incorporates any custom logic you define. This option is best suited for applications that require complex configurations for proper operation, ensuring that related resources with the same lifecycle as the application are deleted when the application is no longer needed.
```

Comment:
The "Clean" button activates a deletion process that includes any custom logic you have defined. This option is most suitable for applications that require complex configurations to function properly. It ensures that any associated resources with the same lifecycle as the application are deleted when the application is no longer needed.